### DISCUSSION TITLE
Disable availability of specific device

### DISCUSSION THREAD

**Discussion by boesing** [Koenkk/zigbee2mqtt#10844] [General]
Hey there,

I've bought the IKEA FLOALT device to lighten my bathroom.
I do not want to enable/disable it via zigbee, the only feature I want to use is that if it becomes available ( due to the power switch), I want to trigger the lights color based on the current time.

- so in the morning, I want to have it super white/bright with all of its power
- starting from noon, I want to have it the "warmer white"
- starting in the late evening (8pm and later) I want to use warm white

This actually works properly but ofc zigbee2mqtt is arguing that the device is not available when its turned off. That is on-purpose tho and I wonder if its possible to disable availability checks and stuff for specific devices (such as my FLOALT panel in the bathroom).

Hope some1 has a good idea.

P.S.: This is not really a problem, when I do not read the log messages, I think that these "failed to ping" messages do not harm anything but I still wonder if its possible to tell zigbee2mqtt that it is fine when devices are not available.

**Comment by sjorge** [comment]
https://www.zigbee2mqtt.io/guide/configuration/device-availability.html#availability-advanced-configuration

> If you want to enable the availability feature for only certain devices, don't add availability: true in your configuration.yaml but specify it for that device only, e.g.
```yml
devices:
  '0x87654321':
    friendly_name: 'my_switch'
    # Enable avaiability for just 'my_switch'
    availability: true
```

You should be able to just set availability to false for your device.

**Reply by boesing** [reply]
Awesome! Will try that. Thanks for pointing me to that config entry.

**Reply by CodeKlopfer** [reply]
Setting "availability: false" in specific devices works for me :)
Would be great to have this option in frontend too.
